govulndb/types.go

```diff
@@ -0,0 +1,48 @@
+package govulndb
+
+import "time"
+
+type Ecosystem string
+type AffectsRangeType int
+
+type Package struct {
+	Name      string
+	Ecosystem Ecosystem
+}
+
+type Affects struct {
+	Ranges []AffectsRange `json:",omitempty"`
+}
+
+type AffectsRange struct {
+	Type       AffectsRangeType
+	Introduced string
+	Fixed      string
+}
+
+type Reference struct {
+	Type string
+	URL  string
+}
+
+type GoSpecific struct {
+	Symbols []string `json:",omitempty"`
+	GOOS    []string `json:",omitempty"`
+	GOARCH  []string `json:",omitempty"`
+	URL     string
+}
+
+type Entry struct {
+	ID         string
+	Published  time.Time
+	Modified   time.Time
+	Withdrawn  *time.Time `json:",omitempty"`
```

Comment:
Why is this a pointer?

Reply:
Source: https://github.com/golang/vulndb/blob/e0c00fae09e687ec6febda47ae3bc7552fc7b988/osv/json.go#L125